apps/creds/tests/test_credentials_request.py

```diff
@@ -0,0 +1,112 @@
+from django.test.client import Client
+from ..models import CredentialingReqest
+from django.urls import reverse
+from apps.test import BaseApiTest
+import datetime
+from rest_framework import exceptions
+from apps.dot_ext.models import Application
+
+
+class CredentialsTestCase(BaseApiTest):
+    def setUp(self):
+        # Test App #1: happy path
+        dev_user1 = self._create_user("developer_test", "123456")
+        test_app1 = self._create_application(
+            "test_app_1", user=dev_user1, data_access_type="THIRTEEN_MONTH"
+        )
+        CredentialingReqest.objects.create(
```

Comment:
Haha, I don't expect you to change this, but this misspelling makes me a do a double take every time.

Reply:
wow, i think my eyes glossed over that until you mentioned it! I can make the change if you and @dtisza1 would like but i'd probably have to change it in a lot of places. Up to you both!

Reply:
@meliGuzman Wow. That misspelling was from a while back. I agree with no need to change it here.